install.sh

```diff
@@ -14,106 +34,173 @@ echo " IP-Address: $IP_RANGE.1"
 echo " IP-Range: $IP_RANGE.0"
 echo " Incomming device: $INC"
 echo " WiFi device: $WIFI"
-
-
-# update os
-apt-get update
-#apt-get -y upgrade
-apt-get -y install hostapd isc-dhcp-server iptables
-
-# check wlan0 available
-
-if ! ifconfig -a | grep "$WIFI"; then
+echo
+
+# Check if INC (eth0) and WIFI (wlan0) are available
+begin "Test if interfaces exist..."
+ifconfig -a | grep "$WIFI" > /dev/null
```

Comment:
don't see the advantage in storing this in `/dev/null` and then getting it via `$?`. Can you explain me whats the advantage here?

Reply:
This avoid to display the standard result of the command on the terminal. Only errors will be displayed.

Reply:
And I prefered to use the $? (which contains the error code of the last executed command) for a better reading of the code.

Reply:
As I use a redirection to avoid displaying info in console, I had to split it into two lines.